wpk/run.sh

```diff
@@ -181,16 +198,25 @@ main() {
         ${PYTHON} /usr/local/bin/wpkpack ${OUTPUT} ${WPKCERT} ${WPKKEY} *
     else
 
-      if [ "${HAVE_PKG_NAME}" == true ]; then
+      if [ "${HAVE_PKG_NAME_WIN}" == true ]; then
           CURRENT_DIR=$(pwd)
           echo "wpkpack ${OUTPUT} ${WPKCERT} ${WPKKEY} ${PKG_NAME} upgrade.bat do_upgrade.ps1"
           cd ${OUTDIR}
           cp ${CURRENT_DIR}/src/win32/{upgrade.bat,do_upgrade.ps1} .
           cp /var/pkg/${PKG_NAME} ${OUTDIR} 2>/dev/null
           wpkpack ${OUTPUT} ${WPKCERT} ${WPKKEY} ${PKG_NAME} upgrade.bat do_upgrade.ps1
           rm -f upgrade.bat do_upgrade.ps1 ${PKG_NAME}
+      elif [ "${HAVE_PKG_NAME_MAC}" == true ]; then
+          CURRENT_DIR=$(pwd)
+          echo "wpkpack ${OUTPUT} ${WPKCERT} ${WPKKEY} ${PKG_NAME} upgrade.sh pkg_installer_mac.sh"
+          cd ${OUTDIR}
+          cp ${CURRENT_DIR}/src/init/pkg_installer_mac.sh .
+          cp ${CURRENT_DIR}/upgrade.sh .
+          cp /var/pkg/${PKG_NAME} ${OUTDIR} 2>/dev/null
+          wpkpack ${OUTPUT} ${WPKCERT} ${WPKKEY} ${PKG_NAME} upgrade.sh pkg_installer_mac.sh
+          rm -f upgrade.sh pkg_installer_mac.sh ${PKG_NAME}
       else
-          echo "ERROR: MSI package is needed to build the Windows WPK"
+          echo "ERROR: a package is needed to build the Windows or macOS WPK"
```

Comment:
Here it would be better to refer to the necessary file instead of saying that a file is needed, like: `MSI/PKG package is needed to build the Windows or macOS WPK`

In principle I suppose that it would be clarified in the documentation but the same script should refer to the necessary package type needed.

Reply:
Done!